wee_alloc/src/lib.rs

```diff
@@ -232,7 +234,10 @@ extern crate unreachable;
 mod extra_assert;
 
 cfg_if! {
-    if #[cfg(target_arch = "wasm32")] {
+    if #[cfg(feature = "static_array_backend")]{
```

Comment:
Nitpick: need a space after `]` and before `{`.

Reply:
👍 Will fix.